### PR TITLE
feat: remove deprecated styles and add stories

### PR DESCRIPTION
**Pull request by NatalieLoyzaga**
## Why

Remove deprecated styles

## What

Remove deprecated styles
Create stories

### CODE REVIEW

File: draft-packages/form/KaizenDraft/Form/Primitives/Label/Label.module.scss [hunk 1]
```diff
@@ -38,15 +43,23 @@ $label-start-margin: 10px;
 // This is to override bootstrap styles. Remove when appropriate
 .text,
 :global(.ideal-sans) .text {
-  @include kz-typography-heading-6;
-  @include ca-inherit-baseline;
+  position: static;
```
Comment (dougmacknz) [inline draft-packages/form/KaizenDraft/Form/Primitives/Label/Label.module.scss:46]: Do you definitely need the `position: static` on these? Would be good to not have that anymore

File: draft-packages/form/docs/Label.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import { ComponentStory, Story } from "@storybook/react"
+import {
+  CATEGORIES,
+  SUB_CATEGORIES,
+  SUB_COMPONENTS_FOLDER_NAME,
+} from "../../../storybook/constants"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.form}/Label`,
```
[the remark this answers is not in the document]
Comment (NatalieLoyzaga) [reply draft-packages/form/docs/Label.stories.tsx:12]: Yeah I had the change originally but then reverted it for some reason - I will add it back

File: draft-packages/form/KaizenDraft/Form/Primitives/Label/Label.module.scss (hunk 1 above)
Comment (NatalieLoyzaga) [reply draft-packages/form/KaizenDraft/Form/Primitives/Label/Label.module.scss:46]: `@include ca-inherit-baseline;` mixin is equal to `position: static;`. I will try removing and run chromatic on it

Comment (NatalieLoyzaga) [reply draft-packages/form/KaizenDraft/Form/Primitives/Label/Label.module.scss]: No chromatic changes :)

**Review by dougmacknz — APPROVED** [review]

**Pull request merged**